src/styles/components/_card-author.scss

```diff
@@ -0,0 +1,13 @@
+@import '../settings/colors';
+
+.w-card-author {
```

Comment:
on mobile we end up with the blue line at the top:

![image](https://user-images.githubusercontent.com/1066253/78311794-b0854900-7506-11ea-9748-9592a0422215.png)


maybe we should always turn that off for author cards.